subsystems/viamserver/viamserver.go

```diff
@@ -160,7 +160,8 @@ func (s *viamServer) Stop(ctx context.Context) error {
 		s.logger.Error(err)
 	}
 
-	if s.waitForExit(ctx, stopTimeout/2) {
+	// 2 min timeout. It must be higher than viam-server shutdown timeout of 90 secs.
```

Comment:
Now that you've changed the TOTAL stopTimeout time (in the manager) to 2 minutes, this needs to be shorter than that by a decent margin, to allow other subsystems time to shutdown as well (they're stopped sequentially.)